admin_manual/release_notes.rst

```diff
@@ -0,0 +1,266 @@
+================================
+ownCloud |version| Release Notes
+================================
+
+Recommended Setup for Running ownCloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ownCloud runs on multiple operating systems and HTTP servers. For best 
+performance, stability, support, and full functionality we recommend:
+
+* Red Hat Enterprise Linux 7
+* MySQL/MariaDB
+* PHP 5.4 +
+* Apache 2.4
+
+Supported Platforms
+^^^^^^^^^^^^^^^^^^^
+
+* Hypervisors: Hyper-V, VMware ESX, Xen, KVM
+* Server: Windows 2008R2; 2012, Linux (Debian 6, RHEL / Centos 6 and 7, Ubuntu 
+  12 and 14, SLES 11, Univention Corporate Server 3.x)
+* Databases: MySQL 5.x; Microsoft SQL Server 2008R2; SQL Server 2012 R2; Oracle 
+  11g
+* Desktop: Windows XP, Windows 7, Mac OS X 10.7+ (64-bit only), Linux (CentOS 
+  6, Ubuntu 10.04 and 11.04+, Fedora 16+, openSUSE 11.4+)
+* Mobile apps: iOS 7+, Android 4+
+* Webserver: IIS (Windows) and Apache2 (Windows and Linux)
+* Web browser: IE8, IE9+, Firefox v14+, Chrome v18+, Safari v5+
```

Comment:
(as we apparently as pointed out in https://github.com/owncloud/documentation/pull/798/files#r23791668 supports only 10.7+)